src/app/shared/drag-and-drop/dragDrop.directive.ts

```diff
@@ -0,0 +1,43 @@
+import { Directive, HostBinding, HostListener, Output, EventEmitter } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+import { FileHandle } from '../../ubs-admin/models/file-handle.model';
+
+@Directive({
+  selector: '[appDrag]'
+})
+export class DragDirective {
+  @Output() files: EventEmitter<FileHandle[]> = new EventEmitter();
+
+  @HostBinding('style.background') public background = '#F7F9FA';
+
+  constructor(private sanitizer: DomSanitizer) {}
+
+  @HostListener('dragover', ['$event']) public onDragOver(evt: DragEvent) {
+    evt.preventDefault();
+    evt.stopPropagation();
+    this.background = '#999';
```

Comment:
please create obj with these colors
like
const color = {
red : '#999'
}

this will be more understandable

Reply:
done